### PR TITLE
travis: wait for slow startup of cert-manager

### PR DESCRIPTION
**Pull request by hunchback**
attempt to fix sporadic failure due to cert-manager not starting up intime

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,9 +26,6 @@ cluster-prepare:
 	$(MAKE) -C third_party/cert-manager deploy
 	$(MAKE) -C third_party/registry deploy
 	$(MAKE) -C third_party/vault deploy
-
-.PHONY: cluster-prepare-wait
-cluster-prepare-wait:
```
Comment (roee88) [inline Makefile:31]: The split (cluster-prepare and cluster-prepare-wait) was done in https://github.com/IBM/the-mesh-for-data/pull/61 by @froesef. It allows installing the integration test stuff while the third parties are being installed.

**Review by roee88 — COMMENTED** [review]
My understanding is that the problem is not that cert-manager isn't ready but rather that we fail to download cert-manager.yaml. See https://github.com/IBM/the-mesh-for-data/issues/88. I don't understand how the changes in this PR help solve the issue. Can you explain?

**Comment by roee88** [comment]
I see now that this is an attempt to solve another problem that isn't described in an issue. So for completeness this is what it tries to solve:
```
Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "webhook.cert-manager.io": Post https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=30s: dial tcp 10.98.47.35:443: connect: connection refused
```

**Comment by hunchback** [comment]
addressing part of https://github.com/IBM/the-mesh-for-data/issues/88

**Comment by hunchback** [comment]
this is going to be redundant due to elimination of stages from travis

**Pull request closed without merging**